natscontext/context.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022-2024 The NATS Authors
+// Copyright 2024 Synadia Communications Inc.
```

Comment:
well - I pulled this code from jsm.go to a large extend? Now I am wondering if that was a good idea.

Reply:
To be double safe - I reverted them to originals.